src/actions/getCurrentVersionByPackage.js

```diff
@@ -9,7 +9,7 @@ export function getCurrentVersionByPackage({ config, npm, props }) {
       .map(name =>
         npm.getCurrentPackageVersion(name).then(version => ({
           name,
-          version,
+          version: version || '1.0.0',
```

Comment:
So `current version` is `null` but evaluated version detects `null` and produces `1.0.0`.